docs/DeveloperGuide.md

```diff
@@ -133,6 +133,25 @@ Classes used by multiple components are in the `seedu.medmoriser.commons` packag
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add Feature (Cheng Jiyuqing)
+
+#### Implementation
+
+This portion describes the implementation of the add feature which allows the user to add new QAndA sets with the question, answer and/or tags.
```

Comment:
I think "and/or" is wrong, because it means answer or tags. I think can change to question, answer and tags (optional).

Reply:
Also I think can remove the 'sets' and just leave it as "add new QAndAs"

Reply:
changed! thanks!